configs/mot/sort/README.md

```diff
@@ -29,9 +29,84 @@ This paper explores a pragmatic approach to multiple object tracking where the m
 
 ## Results and models on MOT17
 
-We implement SORT with independent detector models. To train a model by yourself, you need to train a detector following [here](../../det/).
-The configs in this folder are basically for inference.
-
 | Method |      Detector      | ReID | Train Set  | Test Set | Public | Inf time (fps) | HOTA | MOTA | IDF1 |  FP   |  FN   | IDSw. |                                     Config                                     |                                                       Download                                                       |
 | :----: | :----------------: | :--: | :--------: | :------: | :----: | :------------: | :--: | :--: | :--: | :---: | :---: | :---: | :----------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------------------------: |
 |  SORT  | R50-FasterRCNN-FPN |  -   | half-train | half-val |   N    |      18.6      | 52.0 | 62.0 | 57.8 | 15150 | 40410 | 5847  | [config](sort_faster-rcnn_r50_fpn_8xb2-4e_mot17halftrain_test-mot17halfval.py) | [detector](https://download.openmmlab.com/mmtracking/mot/faster_rcnn/faster-rcnn_r50_fpn_4e_mot17-half-64ee2ed4.pth) |
+
+## Get started
+
+### 1. Training
+
+We implement SORT with independent detector models.
+Note that, due to the influence of parameters such as learning rate in default configuration file,
+we recommend using 8 GPUs for training in order to reproduce accuracy.
+
+You can train the detector as follows.
+
+```shell script
+# Training Faster R-CNN on mot17-half-train dataset with following command.
+# The number after config file represents the number of GPUs used. Here we use 8 GPUs.
+./tools/dist_train.sh \
+    configs/det/faster-rcnn_r50_fpn_8xb2-4e_mot17halftrain_test-mot17halfval.py 8
+```
+
+If you want to know about more detailed usage of `train.py/dist_train.sh/slurm_train.sh`,
+please refer to this [document](../../../docs/en/user_guides/4_train_test.md).
+
+### 2. Testing and evaluation
+
+**2.1 Example on MOTxx-halfval dataset**
+
+```shell script
+# Example 1: Test on motXX-half-val set.
+# The number after config file represents the number of GPUs used. Here we use 8 GPUs.
+./tools/dist_test.sh \
+    configs/mot/sort/sort_faster-rcnn_r50_fpn_8xb2-4e_mot17halftrain_test-mot17halfval.py 8
+```
+
+If you want to use your own detector checkpoint, you can change the cfg as follows.
+
+```shell script
+model = dict(
+    detector=dict(
+        init_cfg=dict(
+            checkpoint=  # noqa: E251
+            'path_to_your_checkpoint.pth'  # noqa: E501
+        )))
+```
+
+Or, you can specify it in commands as follows.
+
+```shell script
+./tools/dist_test.sh \
+    configs/mot/sort/sort_faster-rcnn_r50_fpn_8xb2-4e_mot17halftrain_test-mot17halfval.py 8 \
+    --cfg-options model.detector.init_cfg.checkpoint='path_to_your_checkpoint.pth'
```

Comment:
`--cfg-options model.detector.init_cfg.checkpoint=**path_to_your_checkpoint.pth**`
The quotation marks here should be unnecessary. Do you think it will be misleading to put them here?